### DISCUSSION TITLE
Dark mode

### DISCUSSION THREAD

**Discussion by XarchencoMO** [filamentphp/filament#12294] [Help]
### Package

Form builder

### Package Version

v3.0.0^

### How can we help you?

in filament/forms v2 i could turn off/on dark mode in forms.php file. In filament/forms v3 i added this option in my filament.php file but it doesn't work

**Comment by leandrocfe** [comment]
Add in the `tailwind.config.js`
`darkMode: null`

**Reply by XarchencoMO** [reply]
your solution turns off dark mode in the entire application, i need to turn of dark mode for forms only

**Comment by XarchencoMO** [comment]
hello, @leandrocfe can you help me ?
